packages/calcite-components/src/components/alert/alert.scss

```diff
@@ -4,19 +4,19 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-alert-width: Specifies the width of the component.
+ * @prop --calcite-alert-background-color: Specifies the component's background color.
+ * @prop --calcite-alert-corner-radius: Specifies the component's border radius.
```

Comment:
As a user I might want to override the icon color rendered via prop, or the component's shadow.

Reply:
@alisonailea @jcfranco Should we add tokens for these use cases?

Reply:
Let's discuss customizing the icon and progress color separately. The design spec has both sharing the color defined by `kind`. cc @alisonailea @SkyeSeitz @ashetland